### ISSUE TITLE
Error on run ("Cannot find type")

### ISSUE THREAD

**Issue by baileymh** [RingComics/azuras-star#38]
Loading the exe shows "Cannot find type [system.drawing.icon]: verify that the assembly containing this type is loaded." error message.

Then a series of "You cannot call a method on a null-valued expression."

I have Powershell 7 install, on Windows 10. Running v1.2.2

**Comment by RingComics** [comment]
Odd, this is the first time I've heard of this issue. [system.drawing.icon] should be part of the default system assembly. I'll take a look into importing it redundantly in case something like this happens frequently and isn't an isolated case. I assume this happens every time you launch AS?

**Comment by baileymh** [comment]
It does. I've tried re-downloading, using v1.2.0, restarting, changing the root directory.

Following the error and the series of null-valued expressions, the window opens but it has no size beyond the top toolbar, as if the UI couldn't be created. 

I recently had a power outage, and it had opened fine prior to that, but the program wasn't running when the power went out. I wasn't sure if the powershell somehow got corrupted, so I installed PS 7 manually. But it still did the same thing.
